control/grpc.py

```diff
@@ -828,6 +832,9 @@ def namespace_add_safe(self, request, context):
                     try:
                         ret_del = self.delete_bdev(bdev_name)
                         self.logger.info(f"delete_bdev({bdev_name}): {ret_del.status}")
+                    except AssertionError:
```

Comment:
could you explain AssertionError addition? is not assertion made outside the try .. except block?

Reply:
I added an assert to delete_bdev